test/test-debuglet.ts

```diff
@@ -315,6 +316,49 @@ describe('Debuglet', function() {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should elaborate on inspector warning on 32 bit', function(done) {
```

Comment:
Since the test also tests that the message doesn't appear on 64 bit machines, I think it would be good to add ` but not on 64 bit` to the test name.

Reply:
Done.